### PR TITLE
Add B-SIDES – TRUFFLE AND CHOCOLATE STOUT

### PR DESCRIPTION
**Pull request by iainmullan**

### CODE REVIEW

File: data/truffle_chocolate_stout.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{
+  "name": "B-Sides - Truffle and Chocolate Stout",
```
Comment (GeordieMatt) [inline data/truffle_chocolate_stout.json:2]: I think the convention is to mark it as a B-Side after the title, see the three that I committed the other day.

Comment (sammdec) [inline data/truffle_chocolate_stout.json:2]: Yeah that is what I have started doing for things like the IPA is dead series

**Comment by sammdec** [comment]
You might also want to rebase from master as it looks like there are some conflicts :+1:

**Comment by sammdec** [comment]
@iainmullan I have fixed a couple of the issues and merged your file :+1: Awesome work, thanks for contributing

**Pull request closed without merging**